src/os/OSSelectWakeup.cxx

```diff
@@ -10,7 +10,7 @@ void empty_signal_handler(int)
 #include <sys/stat.h>
 #include <fcntl.h>
 
-#include "../vfs/esp_vfs.h"
+#include <esp_vfs.h>
```

Comment:
are you sure this also compiles under the arduino UI (as opposed to native IDF and whatever build system you use).

Reply:
I tested it on the 1.0.2 arduino-esp32 release and didn't see any failures. I also checked the default include path for 1.0.1 and newer versions and the vfs include directory is present